### DISCUSSION TITLE
Switch Your React App Between Material, Bootstrap and Custom Themes at Runtime

### DISCUSSION THREAD

**Discussion by Shivansh-yadav13** [EddieHubCommunity/LinkFree#740] [Ideas]
- Idea is to switch between custom Primereact themes
- Idea suggested by @schmelto 
- resource https://primetek.hashnode.dev/switch-your-react-app-between-material-bootstrap-and-custom-themes-at-runtime

**Comment by Susmita-Dey** [comment]
It would be better to use Tailwind instead of Bootstrap.

**Comment by Shivansh-yadav13** [comment]
@Susmita-Dey Bootstrap is a PrimeReact theme, current LinkFree is using the SagaBlue theme. You can read the docs for more info.
Example: https://mertsincan.github.io/primereact-dynamic-theming/
